src/components/BookList.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import Card from './Card';
+import AddBook from './AddBook';
+
+function BookList() {
+  const [bookList, setBook] = useState([
+    {
+      id: 1,
+      category: 'Action',
+      name: 'The Hunger Games',
+      director: 'Suzanne Collins',
+      completed: '64%',
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      category: 'Science Fiction',
+      name: 'Dune',
+      director: 'Frank Herbert',
+      completed: '8%',
+      chapter: 'Chapter 3: A Lession',
+    },
+    {
+      id: 3,
+      category: 'Economy',
+      name: 'Capital in the Century',
+      director: 'Suzanne Collins',
+      completed: '0%',
+      chapter: 'Introduction',
+    },
+  ]);
+
+  const addBook = (newBook) => {
+    setBook([newBook, ...bookList]);
+  };
+
+  const handleDelete = (nid) => {
+    setBook(bookList.filter((todo) => todo.id !== nid));
+  };
+
+  return (
+    <div>
+      {bookList.map((todo) => (
+        <Card
+          key={todo.id}
+          category={todo.category}
+          name={todo.name}
+          director={todo.director}
+          completed={todo.completed}
+          chapter={todo.chapter}
+          deleteBook={() => handleDelete(todo.id)}
+        />
+      ))}
+      <div className="border-b-2 border-b-gray-200 pt-5" />
+      <AddBook handleAdd={addBook} />
+      {/* <BookList books={books} /> */}
```

Comment:
👍 Thank you for your hard work! I genuinely appreciate your dedication to learning and improving the project. I understand that there were many changes required to meet the project requirements, and it's entirely normal to take things one step at a time.

- [ ] [Optional] I noticed that there is some commented-out code present in your codebase. It's generally not considered good practice to keep commented code unless there's a specific reason for it. If the commented code is not necessary or is no longer in use, I would recommend completely deleting it to keep your codebase clean and maintainable.